templates/registration/login.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Login</h1>
+  
+  {% if form.errors %}
+    <p style="">Invalid credentials</p>
+  {% endif %}
```

Comment:
Better to say which errors №9 https://uxplanet.org/12-best-practies-for-sign-up-and-login-page-design-69d6cd045cf